src/css/styles/_sections.scss

```diff
@@ -0,0 +1,88 @@
+.section__cover-img {
+  position: relative;
+  width: 100%;
+  height: 360px;
+  overflow: hidden;
+}
+
+.section__title {
+  position: relative;
+  top: 75%;
+  transform: translateY(-60%);
+}
+
+.section__cover-img--naprawa {
+  background: url("../img/naprawa__foto.png") no-repeat 5%;
+  background-size: cover;
+}
+
+.section_texts {
+  padding: 0.5rem;
+}
+
+.section__header {
+  margin: 2rem 0;
+  letter-spacing: 0.1em;
+}
+
+.section__p {
+  margin-bottom: 1.6rem;
+}
+
+.section__p--2 {
+  margin-bottom: 3rem;
+}
+
+.section__p--3 {
+  margin-bottom: 0;
+}
+
+.section__list {
+  padding: 0;
+  list-style: none;
+  counter-reset: list-counter;
+}
```

Comment:
IMHO list is a separate block and this should not be connected with section block

- [ ] list as separate block